src/app/helptext/task-calendar/replication.ts

```diff
@@ -0,0 +1,156 @@
+    netcat_active_side_tooltip: T('System that will open ports for the direct connection'),
+
+    netcat_active_side_listen_address_placeholder: T('Netcat Active Side Listen Address'),
+    netcat_active_side_listen_address_tooltip: T(''),
+
+    netcat_active_side_port_min_placeholder: T('Netcat Active Side Min Port'),
+    netcat_active_side_port_min_tooltip: T('Lowest port number open to connections.'),
+
+    netcat_active_side_port_max_placeholder: T('Netcat Active Side Max Port'),
+    netcat_active_side_port_max_tooltip: T('Highest port number open to connections.'),
+
+    netcat_passive_side_connect_address_placeholder: T('Netcat Active Side Connect Address'),
+    netcat_passive_side_connect_address_tooltip: T(''),
+
+    source_datasets_placeholder: T('Source Datasets'),
+    source_datasets_tooltip: T('Choose one or more datasets on the source system to be replicated.\
+ Each dataset must have an enabled periodic snapshot task.'),
+
+    target_dataset_placeholder: T('Target Datasets'),
+    target_dataset_tooltip: T('Enter the dataset on the remote or destination system where\
+ store snapshots. Example: Poolname/Datasetname, not the mountpoint or filesystem path'),
+
+    recursive_placeholder: T('Recursive'),
+    recursive_tooltip: T('Include all child dataset snapshots in the replication.'),
+
+    exclude_placeholder: T('Exclude Child Datasets'),
+    exclude_tooltip: T('Child datasets to exclude from the recursive replication. Example:\
+ Poolname/Datasetname/childdataset, not the mountpoint or filesystem path.'),
+
+    periodic_snapshot_tasks_placeholder: T('Periodic Snapshot Tasks'),
+    periodic_snapshot_tasks_tooltip: T('Snapshot source for this replication task. Choose from configured\
+ <a href="%%docurl%%/tasks.html#periodic-snapshot-tasks" target="_blank">Periodic Snapshot Tasks</a>.\
+ The replication task must have the same Recursive and Exclude Child Datasets as the chosen periodic snapshot task.'),
+
+    naming_schema_placeholder: T('Naming Schema'),
+    naming_schema_tooltip: T(''),
+
+    also_include_naming_schema_placeholder: T('Also Include Naming Schema'),
+    also_include_naming_schema_tooltip: T(''),
+
+    auto_placeholder: T('Run Automatically'),
+    auto_tooltip: T('Start this replication task immediately after the linked periodic snapshot task completes.'),
+
+    schedule_placeholder: T('Schedule'),
+    schedule_tooltip: T('Define specific times to automatically start the replication task. Disables running the\
+ replication immediately after the periodic snapshot task.'),
+
+    schedule_picker_tooltip: T('Select a schedule preset or choose Custom to open the advanced scheduler.'),
+
+    schedule_begin_placeholder: T('Begin'),
+    schedule_begin_tooltip: T('Define a time to start the replication task.'),
+
+    schedule_end_placeholder: T('End'),
+    schedule_end_tooltip: T('Define the point in time by which replication must start. A started replication\
```

Comment:
Is this correct for **schedule end**?? I'm confused on why it says to specify a "start" time.

Reply:
The logic is reversed - "by which" means the replication cannot start after the defined point in time. Let's rewrite this one the other way 'round:
"Define a time. Replication tasks are not allowed to start after this time. Replications already in progress will continue to run after this time."